content/applications/sales/crm/pipeline/merge_similar.rst

```diff
@@ -0,0 +1,115 @@
+=====================================
+Merge similar leads and opportunities
+=====================================
+
+Odoo automatically detects similar *leads* and *opportunities* within the *CRM* app. Identifying
+these duplicated records allows them to be merged without losing any information in the process.
+Not only does this help keep the *pipeline* organized, but it also prevents customers from being
+contacted by more than one salesperson.
+
+.. note::
+   When merging opportunities, no information is lost. Data from the other opportunity is logged in
+   the chatter, and the information fields, for reference.
+
+Identify similar leads and opportunities
+========================================
+
+Similar leads and opportunities are identified by comparing the *email address* and *phone number*
+of the associated contact. If a similar lead/opportunity is found, a *Similar Leads* smart button
+appears at the top of the lead (or opportunity) record.
+
+.. image:: merge_similar/similar-smart-button.png
+   :align: center
+   :alt: An opportunity record with emphasis on the Similar Leads smart button.
+
+Comparing similar leads and opportunities
+-----------------------------------------
+
+To compare the details of similar leads/opportunities, navigate to :menuselection:`CRM app -->
+Pipeline` or :menuselection:`CRM app --> Leads`. Open a lead or opportunity, and click the
+Doing so opens a Kanban view that only displays similar leads/opportunities. Click on a card to view
+the details for the lead/opportunity, and confirm if they should be merged.
+
+Merging similar leads and opportunities
+=======================================
+
+.. important::
+   When merging, Odoo gives priority to whichever lead/opportunity was created in the system first,
+   merging the information into the first created lead/opportunity. However, if a lead and an
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+After confirming that the leads/opportunities should be merged, return to the Kanban view using the
+breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the :guilabel:`≣
+(view list)` icon to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:guilabel:`⚙️ Actions (gear)` icon at the top of the page, to reveal a drop-down menu. From that
+drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
```

Comment:
Optional, no need to include an icon descriptor when the icon is already accompanied by a label "Actions"
```suggestion
:guilabel:`⚙️ Actions` icon at the top of the page, to reveal a drop-down menu. From that drop-down
menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
```